shuup/customer_group_pricing/models.py

```diff
@@ -42,3 +42,32 @@ def save(self, *args, **kwargs):
         # check if there is a shop product before bumping the cache
         if self.product.shop_products.filter(shop_id=self.shop.id).exists():
             bump_cache_for_product(self.product, self.shop)
+
+
+class CgpDiscount(MoneyPropped, models.Model):
```

Comment:
These to models in this file shares almost every field. You probably should get rid of the duplicate code here.